tests/manager/celery/tasks/test_search.py

```diff
@@ -73,6 +75,36 @@ def test_search_reindex(
     end_to_end_search_fixture.expect_results([work1, work2, work4], "", ordered=False)
 
 
+def test_PP_1332_fiction_returns_results(
+    db: DatabaseTransactionFixture, end_to_end_search_fixture: EndToEndSearchFixture
+) -> None:
+    work1 = db.work(with_open_access_download=True, fiction=True)
+    work2 = db.work(with_open_access_download=True, fiction=False)
+    documents = get_work_search_documents(db.session, 2, 0)
+    assert {doc["_id"] for doc in documents} == {work2.id, work1.id}
+
+    end_to_end_search_fixture.populate_search_index()
+    end_to_end_search_fixture.expect_results(
+        expect=[work1, work2], ordered=False, query_string=""
+    )
+    default_filter = Filter()
```

Comment:
This isn't really the default filter, the default filter is what you get when you don't pass a filter into the query. This should probably be called `filter` or `json_filter`.